Contact/Contact Class and Smaller Related Data Classes/EmailAddress.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Nate.ContactApp
+{
+    public class EmailAddress
+    {
+        private string emailAddress;
```

Comment:
public string Address { get;set; } does the same thing.  There is no reason to do it this way unless you need to apply customer logic to value before assigning and/or returning it.